src/bindings/js/node/src/compiled_model.cpp

```diff
@@ -70,7 +70,7 @@ Napi::Value CompiledModelWrap::get_outputs(const Napi::CallbackInfo& info) {
     auto cm_outputs = _compiled_model.outputs();  // Output<Node>
     Napi::Array js_outputs = Napi::Array::New(info.Env(), cm_outputs.size());
 
-    size_t i = 0;
```

Comment:
What was the issue with `size_t`?

Reply:
This warning interpreted as error. This fix resolves it.
![MicrosoftTeams-image](https://github.com/openvinotoolkit/openvino/assets/72441332/8c15975a-0ec5-45a4-8b99-26acc1ceddf0)